utils.ts

```diff
@@ -1,11 +1,15 @@
 import "allocator/tlsf";
+import { allocateUnsafe, copyUnsafe, HEADER_SIZE } from "internal/string"
```

Comment:
How did you even find out about these functions? Docs are verrrry hard to work with at this point heh? :(